### ISSUE TITLE
Feature request: Color contrast ratio report

### ISSUE THREAD

**Issue by jkotlinski** [jkotlinski/lsdpatch#77]
In the World Wide Web, Contrast Ratio is sometimes used as a guideline for picking text foreground/background colors. As an example, using color contrast ratio 3 or above, should be totally safe for most people.

It could be interesting to have an option to generate a color contrast ratio report in the Palette editor, perhaps as a menu option.

More info: https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html
